tests/tape/test_jacobian_tape.py

```diff
@@ -838,6 +839,30 @@ def __add__(self, other):
             def __radd__(self, other):
                 return self + other
 
+            def __truediv__(self, other):
```

Comment:
How come all these methods were added?

Reply:
I extended the test to the finite-diff transform and it was complaining that these operators were not defined. This test is covering this code line which I had to modify to not stack a scalar (bug):
https://github.com/PennyLaneAI/pennylane/blob/38c2dd3a8c365859d43cc41f933d1a66dc045670/pennylane/gradients/parameter_shift.py#L278-L280

Reply:
With the jacobian tape removed I moved these tests to the corresponding `tests/gradients/*` tests and simplified them.